### PR TITLE
PersistentVolumeClaim details pane now shows a link to it's PersistVolume (if any)

### PR DESCRIPTION
**Pull request by marcosdiez**
image

### CODE REVIEW

**Comment by maciaszczykm** [comment]
LGTM but there is a conflict.

**Comment by marcosdiez** [comment]
Hello @maciaszczykm ! The conflict is gone (it was only translations)

File: i18n/messages.xlf [hunk 1]
```diff
@@ -742,6 +742,170 @@
           <context context-type="linenumber">66</context>
         </context-group>
       </trans-unit>
+<<<<<<< HEAD
```
Comment (maciaszczykm) [inline i18n/messages.xlf:745]: Some merge issue.

**Comment by marcosdiez** [comment]
my bad. now it's *really* fixed.

**Comment by marcosdiez** [comment]
Hey, @maciaszczykm  
I was thinking, most of my commits is me fighting with i18n stuff (that are unrelated to my code)
that also makes the code review much harder (because the i18n stuff is only noise for code review)
and it also makes impossible to merge two UI related PRs at once.

Does it still make sense for us to include i18n stuff in our feature PRs ?

**Comment by maciaszczykm** [comment]
Yes, the process is a bit broken right now but it has to be done regularly.

**Comment by jeefy** [comment]
@marcosdiez Looks like e2e tests are failing :( Can you address?

**Comment by marcosdiez** [comment]
Hello @jeefy 
This is a problem in the test, not in my code.
The proof for it is that my last real commit (with code, not i18n stuff) worked:

image

So if you *really* want, I can keep sending empty commit. Eventually all tests will pass. But there is nothing for me to do on my side...

**Pull request merged**